cypress/integration/HomeworkTests/calculator.spec.js

```diff
@@ -0,0 +1,231 @@
+/// <reference types="cypress" />
+import HomeworkPage from '../../page_objects/HomeworkPage/HomeworkPage.js'
+
+var homeworkPage = new HomeworkPage
+
+const builds = ['Prototype', 1, 2, 3, 4, 5, 6, 7, 8, 9]
+
+// 0 - not checked ; 1 - checked 
+const integersOnly = [0, 1]
+
+beforeEach('Executes before each test', ()=>{
+    cy.visit('https://testsheepnz.github.io/BasicCalculator')
+}) 
+
+//ADD
+integersOnly.forEach((integer) => {
+    builds.forEach((build) =>{
+        it(`Adds two numbers correctly for ${build} build, integers only = ${integer}`, ()=>{  
+            if(integer == 1){
+                homeworkPage.getIntegersOnlyCheckbox().click();
+            }
+            homeworkPage.getBuildDropdown().select(`${build}`)  
+            homeworkPage.getNumberOneField().type(3.5)
+            homeworkPage.getNumberTwoField().type(4)
+            homeworkPage.getCalculateButton().click()
+            if(integer == 1){
+                homeworkPage.getAnswerField().should('have.value', '7')     
+            }else{
+                homeworkPage.getAnswerField().should('have.value', '7.5') 
+            }               
+        })
+    })
+})
+
+//SUBTRACT
+integersOnly.forEach((integer) => {
+    builds.forEach((build) =>{
+        it.only(`Subtracts two numbers correctly for ${build} build, integers only = ${integer}`, ()=>{
+            if(integer == 1){
+                homeworkPage.getIntegersOnlyCheckbox().click();
+            }
+            homeworkPage.getBuildDropdown().select(`${build}`)
```

Comment:
Inconsistent use of quotes (sometimes you use ' and sometimes ` )

Reply:
${build} didn't work with normal quotes('') for some reason, maybe just javascript syntax feature, so had to use `` which i would never use otherwise...